Facebook-DP-Downloader/fb_dp_downloader.py

```diff
@@ -0,0 +1,40 @@
+"""
+    Facebook-DP-Downloader
+        Download the profile picture of any public profile on Facebook
+        by just having it's Facebook id 
+    
+"""
+
+import os
+import requests
+
+url="https://graph.facebook.com/{}/picture?type=large"
+
+""" This url is the url provided by the Facebook graph api
+which helps to get to the profile picture of the corresponding Facebook id 
+{}==Facebook id 
+Facebook id  denotes the unique user id of the Facebook profile
+whose profile we are requesting
+"""
+
+path = os.getcwd()
+# get the path of the current working directory
+
+if not "fb_dps" in os.listdir(path):
+    os.mkdir("fb_dps")
+
+"""checks if the folder exists in the current working directory.
+If it does not exist, then it gets created
+"""
+
+fbid=int(input("Enter the initial Facebook-id to download profile picture: "))
+# range here can vary, but the number should be a valid Facebook id 
+
+try:
+    result=requests.get(url.format(fbid))
+    with open("fb_dps/{}_img.jpg".format(fbid),"wb") as file:
+        file.write(result.content)
+
+except OAuthException as e:
+	print("An access token is required to request this resource.")
+
```

Comment:
Also, I tried downloading some pictures, some files were also shown but they didn't opened, showed the error. I tried the number `4` and it downloaded that only. Except this, none of the files were downloaded.

Reply:
The profile pictures will only get downloaded if the respective Facebook profile exists for the corresponding Facebook user id entered (the profile does may/may not exist for every numerical value) , and if exists then, is public along with no profile picture guard set. 
Also no pictures will get downloaded for Facebook ids below 4, since there exists no such id.

Reply:
okay, got it. Make the changes suggested excluding this

Reply:
Okay 👍